src/Bullet/App.php

```diff
@@ -59,7 +59,8 @@ function($response) {
             function($response) {
                 $response->contentType('application/json');
                 $response->content(json_encode($response->content()));
-            }
+            },
+            'array-content'
```

Comment:
True, but if they are significantly changing the behavior, then I would say they should just remove it and replace it with something else.